docs/Makefile

```diff
@@ -0,0 +1,189 @@
+# Makefile for Sphinx documentation
```

Comment:
Is this file generated by the Sphinx tool?

Reply:
Tiny file is generated, for enabling more options like

- generate offline PDF
- generate offline HTML
- generate offline latex
- generate offline xml
- geneate offline man pages
- generate offline info
- generate offline json

I added few more options.